.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2
 jobs:
-  build:
+  build_docs:
```

Comment:
BTW I pushed a commit to `circleci-artifacts-redirector` that made it so that `build` is okay, too. But `build_docs` might be a clearer name anyway.

Reply:
Thanks!